packages/format-library/src/text-color/index.js

```diff
@@ -15,10 +15,10 @@ import { removeFormat } from '@wordpress/rich-text';
 /**
  * Internal dependencies
  */
-import { default as InlineColorUI, getActiveColor } from './inline';
+import { default as InlineColorUI, getActiveColors } from './inline';
 
-const name = 'core/text-color';
-const title = __( 'Text color' );
+const name = 'core/color';
```

Comment:
It would be good to double-check if `core/text-color` isn't used by plugins to modify this formatting control. Using a different name, as much as it makes sense in this context, it looks like a breaking change at first glance.

Reply:
Restored it.